lib/Newsletter/NewsletterSaveController.php

```diff
@@ -291,4 +276,31 @@ private function rescheduleIfNeeded(NewsletterEntity $newsletter, Newsletter $ne
       $this->entityManager->refresh($newsletterOption);
     }
   }
+
+  private function updateQueue(NewsletterEntity $newsletter, Newsletter $newsletterModel, array $options) {
+    $queue = $newsletter->getLatestQueue();
+    if (!$queue) {
+      return;
+    }
+
+    if (in_array($newsletter->getType(), [NewsletterEntity::TYPE_NOTIFICATION, NewsletterEntity::TYPE_NOTIFICATION_HISTORY], true)) {
```

Comment:
I know it is a reimplementation of current state but this condition makes me think. Do we need to do this for other automatic emails like welcome emails or WooCommerce emails? This will regenerate the body and subject of the last sent email. Maybe we even don't want to do it for welcome emails and WooCommerce emails.

Reply:
@costasovo In other words, we should regenerate the body and subject only for `standard` newsletters, right? (All types are `automatic`, `standard`, `welcome`, `notification`, `notification_history`, `wc_transactional`.)

Reply:
Hey. I would say so.